src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, removeBook }) => {
```

Comment:
You could change this to `handleRemoveBook`.
```suggestion
const Book = ({ book, handleRemoveBook }) => {
```